tests/settings.py

```diff
@@ -122,7 +125,7 @@ def userdatas(self) -> List[UserData]:
         userinfos = self.userinfos
         lab_statuses = [x for x in LabStatus]
         pod_states = [x for x in PodState]
-        for idx, v in enumerate(userinfos):
+        for idx, v in enumerate(userinfos.values()):
```

Comment:
Was this wrong before, but accidentally passing?

Reply:
No, it was correct, I changed the list to a dict because I needed the names of the tokens to reuse the same data structure for the Gafaelfawr mock. So that required this change to get back the original list.

Reply:
Oh, OK, I see now how it's coupled to the next bit.